python/cudf/cudf/core/index.py

```diff
@@ -311,6 +311,10 @@ def __contains__(self, item):
             item, tuple(np.sctypes["int"] + np.sctypes["float"] + [int, float])
         ):
             return False
+        try:
+            item = pd.core.dtypes.common.ensure_python_int(item)
```

Comment:
`ensure_python_int` only really calls `int(item)` in essence. Maybe to avoid the private pandas import `int` can just be used and catch the `TypeError` and `ValueError`?

Reply:
I was under an assumption this was a public method, I'm trying to avoid the following:
```python
In [1]: int(True)
Out[1]: 1
```

Reply:
Ah I see. The `pandas.core` namespace is considered "private" (things could get moved around without warning), but I suppose this is okay to use. `dtypes/common.py` doesn't tend to change much